orchagent/aclorch.cpp

```diff
@@ -4467,7 +4471,7 @@ void AclOrch::registerFlexCounter(const AclRule& rule)
 void AclOrch::deregisterFlexCounter(const AclRule& rule)
 {
     auto ruleIdentifier = generateAclRuleIdentifierInCountersDb(rule);
-    m_countersDb.hdel(COUNTERS_ACL_COUNTER_RULE_MAP, rule.getId());
+    m_countersDb.hdel(COUNTERS_ACL_COUNTER_RULE_MAP, ruleIdentifier);
```

Comment:
@vadymhlushko-mlnx why do we need this one?

Reply:
Because we pass to the `hdel` function invalid parameter and after deleting the PBH rule the `ACL_COUNTER_RULE_MAP` key was presented in COUNTERS_DB.